### ISSUE TITLE
when running another mpv instance

### ISSUE THREAD

**Issue by phanirithvij** [Francesco149/weebp#14]
When running another mpv instance before starting 
```shell
wp run mpv ....
```
wp will push the existing mpv into the background instead of opening the new one in the background.

[A video showing what I mean.](https://res.cloudinary.com/rootworld/video/upload/v1585047858/Weebp-Mpv.mp4)

**Comment by phanirithvij** [comment]
contents of `videowalls.bat`

```bat
wp run mpv ^
--player-operation-mode=pseudo-gui ^
--force-window=yes ^
--terminal=no ^
--no-audio ^
--sub=no ^
--loop=inf ^
--fullscreen ^
--input-ipc-server=\\.\pipe\mpvsocket ^
"D:\Images\VideoWallpapers"

wp mv --wait --class mpv -x 1920
wp add --wait --fullscreen --class mpv

```

**Comment by Francesco149** [comment]
yeah, you can't use class with multiple mpv's. you could set the window title by passing --title="something" to mpv and then using the window name to find it

**Comment by phanirithvij** [comment]
So I was able to do it like this

```bat
wp run mpv ^
--player-operation-mode=pseudo-gui ^
--force-window=yes ^
--terminal=no ^
--title=wp ^
--no-audio ^
--sub=no ^
--loop=inf ^
--fullscreen ^
--input-ipc-server=\\.\pipe\mpvsocket ^
"D:\Images\VideoWallpapers"

wp mv --wait --name wp -x 1920
wp add --wait --fullscreen --name wp
```

How can I kill that specific mpv instance?

Right now I'm doing
```bat
taskkill /im mpv.exe /f
wp.exe killall
```
Which kills all the running mpv instances.

**Comment by Francesco149** [comment]
`wp kill --name wp` or `wp killall --name wp`

**Comment by phanirithvij** [comment]
It doesn't work.
The mpv is still there. One more thing, I don't see wp in task manager when it is running 

![image](https://user-images.githubusercontent.com/29627898/81418234-fd91b600-9169-11ea-860d-8f58c4bc485d.png)

**Comment by phanirithvij** [comment]
Got something working to kill that wallpaper mpv instance (terrible hack sorry)
```bat
set cmd=powershell -c "get-process mpv | where-object {$_.mainwindowhandle -eq 0} | select-object Id"
:: https://stackoverflow.com/a/27429851/8608146
for /F "delims=" %%a in ('%cmd%') do (
   set "lastButOne=!lastLine!"
   set "lastLine=%%a"
)
taskkill /PID %lastLine% /f
REM timeout 5000

:: This does nothing (??)
wp.exe killall --name wp
```

**Comment by Francesco149** [comment]
hmm that's strange. `wp kill` and `wp killall` both find the window in the same way `wp mv` does. so I don't see how it wouldn't find it. I'll look into this when i have time I guess

**Comment by phanirithvij** [comment]
The title of mpv is getting removed when it is set as a wallpaper for some reason. That's why I was able to filter that mpv process by no title and get the pid.

**Comment by phanirithvij** [comment]
Btw I am once again asking for the battlefield video wallpaper that's in th readme

**Comment by Francesco149** [comment]
that makes sense. but how does the mv command work then? hmm

well if the window can't retain its title it's gonna be hard to make wp kill work

and I don't have that wallpaper anymore
